Doc/howto/descriptor.rst

```diff
@@ -934,32 +934,51 @@ here is a pure Python equivalent:
             if doc is None and fget is not None:
                 doc = fget.__doc__
             self.__doc__ = doc
+            self.name = None
+
+        def __set_name__(self, owner, name):
+            self.name = name
 
         def __get__(self, obj, objtype=None):
             if obj is None:
                 return self
             if self.fget is None:
-                raise AttributeError("unreadable attribute")
+                raise AttributeError(
```

Comment:
Let's get this back to a single line:

raise AttributeError(f'unreadable attribute {self._name}')

Also, please use single quotes to match the style used elsewhere in the document.

Reply:
Changed, thanks